pyproject.toml

```diff
@@ -7,6 +7,9 @@ license = "MIT"
 
 [tool.poetry.dependencies]
 python = "3.10.*"
+pycryptodome = "^3.15.0"
+setuptools = "^63.2.0"
```

Comment:
Since we won't be using the pqcrypto library after all afaik are setup tools still necessary?